.github/workflows/test-security-hub-scan.yaml

```diff
@@ -0,0 +1,47 @@
+name: Test security-hub-scan
+
+on:
+  workflow_call: ~
+
+
+jobs:
+  run-test:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      id-token: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+      # run setup to get uds to use zarf
+      - name: Environment setup
+        uses: ./.github/actions/setup
+
+      - name: build example package
+        id: setup
+        run: |
+          uds run create-nginx-package --no-progress
+          echo "path="$(ls *.zst) >> "$GITHUB_OUTPUT"
+        shell: bash
+
+      - name: run scan
+        id: scan
+        uses: ./.github/actions/security-hub-scan
+        with:
+          path: ${{ steps.setup.outputs.path }}
+
+      # verify that the outputs and the cosign bundle are correct
+      - name: install cosign
+        uses: sigstore/cosign-installer@4959ce089c160fddf62f7b42464195ba1a56d382 # v3.6.0
```

Comment:
given this takes 2s it should likely be simplified to happen in the env setup (we will want to use cosign for package / bundle signing later too)

Reply:
added to the setup step